src/framework/components/light/component.js

```diff
@@ -1139,6 +1180,25 @@ class LightComponent extends Component {
         return this.light.penumbraSize;
     }
 
+    /**
+     * Sets the falloff rate for shadow penumbra for contact hardening shadows.
+     * Defaults to 1.
```

Comment:
What's the range? And how does the falloff behave exactly? Maybe a few more words here.